pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -519,7 +519,7 @@ func (ctrl *PersistentVolumeController) syncVolume(volume *v1.PersistentVolume)
 		if err != nil {
 			return err
 		}
-		if !found && metav1.HasAnnotation(volume.ObjectMeta, pvutil.AnnBoundByController) {
```

Comment:
The original codes seem not consistent with the following comments.

Is `!found && !metav1.HasAnnotation(volume.ObjectMeta, pvutil.AnnBoundByController)` right ?